emirge missing packages on macOS

I am trying to install emirge on the new M1 MacBook Pro, OS Big Sur 11.2.3. When attempting to run the installation, I get the following terminal output (posting full install output to cover all bases):

```
#####################################################
# This script installs mirgecom, and dependencies.  #
#####################################################

==== Conda installation
==== Installing Miniforge in /Users/dshtey2/Documents/MIRGE/emirge/miniforge3.
PREFIX=/Users/dshtey2/Documents/MIRGE/emirge/miniforge3
Unpacking payload ...

                                           __
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/


Transaction

  Prefix: /Users/dshtey2/Documents/MIRGE/emirge/miniforge3

  Updating specs:

   - python==3.9.2=hcbd9b3a_0_cpython
   - bzip2==1.0.8=h27ca646_4
   - ca-certificates==2020.12.5=h4653dfc_0
   - libcxx==11.0.1=h168391b_0
   - ncurses==6.2=h9aa5885_4
   - tzdata==2021a=he74cb21_0
   - xz==5.2.5=h642e427_1
   - yaml==0.2.5=h642e427_0
   - zlib==1.2.11=h31e879b_1009
   - libffi==3.3=h9f76cd9_2
   - openssl==1.1.1j=h27ca646_0
   - readline==8.0=hc8eb9b7_2
   - tk==8.6.10=hf7e6567_1
   - sqlite==3.34.0=h6d56c25_0
   - idna==2.10=pyh9f0ad1d_0
   - pycparser==2.20=pyh9f0ad1d_2
   - python_abi==3.9=1_cp39
   - six==1.15.0=pyh9f0ad1d_0
   - tqdm==4.57.0=pyhd8ed1ab_0
   - wheel==0.36.2=pyhd3deb0d_0
   - certifi==2020.12.5=py39h2804cbe_1
   - cffi==1.14.5=py39h702c04f_0
   - chardet==4.0.0=py39h2804cbe_1
   - conda-package-handling==1.7.2=py39h51e6412_0
   - pycosat==0.6.3=py39h46acfd9_1006
   - pysocks==1.7.1=py39h2804cbe_3
   - ruamel_yaml==0.15.80=py39h46acfd9_1004
   - brotlipy==0.7.0=py39h46acfd9_1001
   - cryptography==3.4.4=py39h6e07874_0
   - setuptools==49.6.0=py39h2804cbe_3
   - pip==21.0.1=pyhd8ed1ab_0
   - pyopenssl==20.0.1=pyhd8ed1ab_0
   - urllib3==1.26.3=pyhd8ed1ab_0
   - requests==2.25.1=pyhd3deb0d_0
   - conda==4.9.2=py39h2804cbe_0


  Package                   Version  Build               Channel       Size
─────────────────────────────────────────────────────────────────────────────
  Install:
─────────────────────────────────────────────────────────────────────────────

  brotlipy                    0.7.0  py39h46acfd9_1001   installed         
  bzip2                       1.0.8  h27ca646_4          installed         
  ca-certificates         2020.12.5  h4653dfc_0          installed         
  certifi                 2020.12.5  py39h2804cbe_1      installed         
  cffi                       1.14.5  py39h702c04f_0      installed         
  chardet                     4.0.0  py39h2804cbe_1      installed         
  conda                       4.9.2  py39h2804cbe_0      installed         
  conda-package-handling      1.7.2  py39h51e6412_0      installed         
  cryptography                3.4.4  py39h6e07874_0      installed         
  idna                         2.10  pyh9f0ad1d_0        installed         
  libcxx                     11.0.1  h168391b_0          installed         
  libffi                        3.3  h9f76cd9_2          installed         
  ncurses                       6.2  h9aa5885_4          installed         
  openssl                    1.1.1j  h27ca646_0          installed         
  pip                        21.0.1  pyhd8ed1ab_0        installed         
  pycosat                     0.6.3  py39h46acfd9_1006   installed         
  pycparser                    2.20  pyh9f0ad1d_2        installed         
  pyopenssl                  20.0.1  pyhd8ed1ab_0        installed         
  pysocks                     1.7.1  py39h2804cbe_3      installed         
  python                      3.9.2  hcbd9b3a_0_cpython  installed         
  python_abi                    3.9  1_cp39              installed         
  readline                      8.0  hc8eb9b7_2          installed         
  requests                   2.25.1  pyhd3deb0d_0        installed         
  ruamel_yaml               0.15.80  py39h46acfd9_1004   installed         
  setuptools                 49.6.0  py39h2804cbe_3      installed         
  six                        1.15.0  pyh9f0ad1d_0        installed         
  sqlite                     3.34.0  h6d56c25_0          installed         
  tk                         8.6.10  hf7e6567_1          installed         
  tqdm                       4.57.0  pyhd8ed1ab_0        installed         
  tzdata                      2021a  he74cb21_0          installed         
  urllib3                    1.26.3  pyhd8ed1ab_0        installed         
  wheel                      0.36.2  pyhd3deb0d_0        installed         
  xz                          5.2.5  h642e427_1          installed         
  yaml                        0.2.5  h642e427_0          installed         
  zlib                       1.2.11  h31e879b_1009       installed         

  Summary:

  Install: 35 packages

  Total download: 0  B

─────────────────────────────────────────────────────────────────────────────



Transaction starting
Linking bzip2-1.0.8-h27ca646_4
Linking ca-certificates-2020.12.5-h4653dfc_0
Linking libcxx-11.0.1-h168391b_0
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libc++.1.0.dylib: replacing existing signature
Linking ncurses-6.2-h9aa5885_4
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libncurses.6.dylib: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libncursesw.6.dylib: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libtinfo.6.dylib: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libtinfow.6.dylib: replacing existing signature
Linking tzdata-2021a-he74cb21_0
Linking xz-5.2.5-h642e427_1
Linking yaml-0.2.5-h642e427_0
Linking zlib-1.2.11-h31e879b_1009
Linking openssl-1.1.1j-h27ca646_0
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/bin/openssl: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/engines-1.1/capi.dylib: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/engines-1.1/padlock.dylib: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libcrypto.1.1.dylib: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libssl.1.1.dylib: replacing existing signature
Linking libffi-3.3-h9f76cd9_2
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libffi.7.dylib: replacing existing signature
Linking readline-8.0-hc8eb9b7_2
Linking tk-8.6.10-hf7e6567_1
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libtcl8.6.dylib: replacing existing signature
Linking sqlite-3.34.0-h6d56c25_0
Linking python-3.9.2-hcbd9b3a_0_cpython
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/bin/python3.9: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/libpython3.9.dylib: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_asyncio.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_bisect.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_blake2.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_bz2.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_codecs_cn.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_codecs_hk.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_codecs_iso2022.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_codecs_jp.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_codecs_kr.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_codecs_tw.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_contextvars.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_crypt.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_csv.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_ctypes.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_ctypes_test.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_curses.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_curses_panel.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_datetime.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_dbm.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_decimal.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_elementtree.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_hashlib.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_heapq.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_json.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_lsprof.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_lzma.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_md5.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_multibytecodec.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_multiprocessing.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_opcode.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_pickle.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_posixshmem.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_posixsubprocess.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_queue.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_random.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_scproxy.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_sha1.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_sha256.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_sha3.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_sha512.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_socket.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_sqlite3.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_ssl.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_statistics.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_struct.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_testbuffer.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_testcapi.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_testimportmultiple.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_testinternalcapi.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_testmultiphase.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_tkinter.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_uuid.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_xxsubinterpreters.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_xxtestfuzz.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/_zoneinfo.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/array.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/audioop.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/binascii.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/cmath.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/fcntl.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/grp.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/math.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/mmap.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/nis.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/parser.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/pyexpat.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/readline.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/resource.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/select.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/syslog.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/termios.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/unicodedata.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/xxlimited.cpython-39-darwin.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/lib-dynload/zlib.cpython-39-darwin.so: replacing existing signature
Linking wheel-0.36.2-pyhd3deb0d_0
Linking tqdm-4.57.0-pyhd8ed1ab_0
Linking six-1.15.0-pyh9f0ad1d_0
Linking python_abi-3.9-1_cp39
Linking pycparser-2.20-pyh9f0ad1d_2
Linking idna-2.10-pyh9f0ad1d_0
Linking ruamel_yaml-0.15.80-py39h46acfd9_1004
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/site-packages/ruamel_yaml/ext/_ruamel_yaml.cpython-39-darwin.so: replacing existing signature
Linking pysocks-1.7.1-py39h2804cbe_3
Linking pycosat-0.6.3-py39h46acfd9_1006
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/site-packages/pycosat.cpython-39-darwin.so: replacing existing signature
Linking conda-package-handling-1.7.2-py39h51e6412_0
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/site-packages/conda_package_handling/archive_utils_cy.cpython-39-darwin.so: replacing existing signature
Linking chardet-4.0.0-py39h2804cbe_1
Linking certifi-2020.12.5-py39h2804cbe_1
Linking cffi-1.14.5-py39h702c04f_0
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/site-packages/_cffi_backend.cpython-39-darwin.so: replacing existing signature
Linking setuptools-49.6.0-py39h2804cbe_3
Linking cryptography-3.4.4-py39h6e07874_0
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/site-packages/cryptography/hazmat/bindings/_openssl.abi3.so: replacing existing signature
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/site-packages/cryptography/hazmat/bindings/_padding.abi3.so: replacing existing signature
Linking brotlipy-0.7.0-py39h46acfd9_1001
/Users/dshtey2/Documents/MIRGE/emirge/miniforge3/lib/python3.9/site-packages/brotli/_brotli.abi3.so: replacing existing signature
Linking pip-21.0.1-pyhd8ed1ab_0
Linking pyopenssl-20.0.1-pyhd8ed1ab_0
Linking urllib3-1.26.3-pyhd8ed1ab_0
Linking requests-2.25.1-pyhd3deb0d_0
Linking conda-4.9.2-py39h2804cbe_0
Transaction finished
installation finished.
==== Fetching mirgecom
Cloning into 'mirgecom'...
remote: Enumerating objects: 202, done.
remote: Counting objects: 100% (202/202), done.
remote: Compressing objects: 100% (144/144), done.
remote: Total 6873 (delta 118), reused 108 (delta 56), pack-reused 6671
Receiving objects: 100% (6873/6873), 1.53 MiB | 7.25 MiB/s, done.
Resolving deltas: 100% (4831/4831), done.
==== Create ceesd conda environment
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - pymetis
  - clinfo
```

To try to install these packages I had to install my own local copy of anaconda, but even then the packages were "not available from current channels."

It seems that I am having trouble with the MPI packages. I installed mpi4py, mpich, and open-mpi, but even with all of them installed I get the following error:
``` 
    undef: _MPI_Init 
    undef: _MPI_Finalize
    Undefined symbols for architecture arm64:
      "_MPI_Init", referenced from:
          _main in lto.o
      "_MPI_Finalize", referenced from:
          _main in lto.o
    ld: symbol(s) not found for architecture arm64
    clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
    failure.
    removing: _configtest.c _configtest.o
    error: Cannot link MPI programs. Check your configuration!!!
```

--- a/install-conda.sh
+++ b/install-conda.sh
@@ -12,7 +12,7 @@ MY_CONDA_DIR=${MY_CONDA_DIR:-$HOME/miniforge3}
 
 if [[ ! -d $MY_CONDA_DIR || ! -x $MY_CONDA_DIR/bin/conda ]]; then
     echo "==== Installing Miniforge in $MY_CONDA_DIR."
-    wget -c --quiet https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$myos-"$myarch.sh"
+    curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$myos-"$myarch.sh"
     bash Miniforge3-$myos-"$myarch.sh" -u -b -p "$MY_CONDA_DIR"
 else
     echo "==== Conda found in $MY_CONDA_DIR, skipping Miniforge installation."

--- a/install.sh
+++ b/install.sh
@@ -147,8 +147,8 @@ fi
 # Required to use pocl on macOS Big Sur
 # (https://github.com/illinois-ceesd/emirge/issues/114)
 if [[ $(uname) == "Darwin" ]]; then
-  $(uname -m) == "x86_64" ]] && conda install --yes clang_osx-64
-  $(uname -m) == "arm64" ]] && conda install --yes clang_osx-arm64
+  [[ $(uname -m) == "x86_64" ]] && conda install --yes clang_osx-64
+  [[ $(uname -m) == "arm64" ]] && conda install --yes clang_osx-arm64
 fi
 
 # Install an environment activation script

--- a/install.sh
+++ b/install.sh
@@ -146,8 +146,10 @@ fi
 
 # Required to use pocl on macOS Big Sur
 # (https://github.com/illinois-ceesd/emirge/issues/114)
-[[ $(uname) == "Darwin" && $(uname -m) == "x86_64" ]] && conda install --yes clang_osx-64
-
+if [[ $(uname) == "Darwin" ]]; then
+  $(uname -m) == "x86_64" ]] && conda install --yes clang_osx-64
+  $(uname -m) == "arm64" ]] && conda install --yes clang_osx-arm64
+fi
 
 # Install an environment activation script
 rm -rf "$mcprefix"/config

